### ISSUE TITLE
R101 ImageNet Pretrained model

### ISSUE THREAD

**Issue by tonysy** [facebookresearch/SlowFast#107]
Hi, could you release the resnet-101 imagenet pretrained model for research purpose, thanks a lot.

**Comment by tonysy** [comment]
@takatosp1 Hi, could you share the recipe for training the backbone(res101/res18/res34)? Or can I convert the checkpoints provided in the `torchvision`  for facilitating the research? Thanks.

**Comment by takatosp1** [comment]
Hi, 
Thanks for playing with PySlowFast, we currently has no playing releasing these models but I'll label this issue as enhancement.
Will share the update with you once we release it.
